src/Sylius/Bundle/ShopBundle/Resources/views/Product/_simpleList.html.twig

```diff
@@ -0,0 +1,13 @@
+<div class="ui four column stackable grid">
+    <div class="row">
+        {% for product in products %}
+            <div class="column">
+                {% include '@SyliusShop/Product/_simpleBox.html.twig' %}
+            </div>
+    {% if 0 == loop.index % 4 %}
+    </div>
+    <div class="row">
+    {% endif %}
```

Comment:
I am still on the edge about these, because this version has correct indent according to html structure. It closes div on this level and opens a new row. So I'd keep it that way to make it clear what is going on here in terms for generate html. If we indent it - can be confusing.